src/calculateRentalCost.js

```diff
@@ -28,7 +28,28 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
```

Comment:
you don't need this variable, just return the result immediately instead of this assignment